CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
How does this file / VERSION get updated? Does the release process need to be updated to include entering notes in this changelog (not sure the release process for this repo is even currently documented anywhere)?